### PR TITLE
util:create a test for MultiChildLoadBalancer and Endpoint

### PR DESCRIPTION
**Pull request by larry-safran**
Create a unit test for basic functionality of both MultiChildLoadBalancer and Endpoint.

### CODE REVIEW

File: util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+  }
+  
+  private LoadBalancer.Subchannel getSubchannel(EquivalentAddressGroup removedEag) {
+    return subchannels.get(Collections.singletonList(removedEag));
+  }
+
+  private static List<Object> getChildEags(MultiChildLoadBalancer loadBalancer) {
+    return loadBalancer.getChildLbStates().stream()
+        .map(ChildLbState::getEag)
+        // .map(EquivalentAddressGroup::getAddresses)
```
Comment (YifeiZhuang) [inline util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java:331]: remove

**Review by YifeiZhuang — APPROVED** [review]
please address comment before merge

File: util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+        createEndpoint(Attributes.EMPTY, "addr1", "addr2"),
+        createEndpoint(Attributes.EMPTY, "addr1"));
+  }
+
+  private String addressesOnlyString(EquivalentAddressGroup eag) {
+    if (eag == null) {
+      return null;
+    }
+
+    System.out.println(eag.getAddresses().toString());
```
[the remark this answers is not in the document]
Comment (larry-safran) [reply util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java]: done

File: util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java (hunk 1 above)
Comment (larry-safran) [reply util/src/test/java/io/grpc/util/MultiChildLoadBalancerTest.java]: done

**Pull request merged**